Makefile

```diff
@@ -42,6 +42,17 @@ parser:
 
 	golex -o parser/scanner.go parser/scanner.l
 
+
+check:
+	@echo "vet"	
+	@! go tool vet . 2>&1 | grep -vE 'Godeps|unreachable code'
```

Comment:
@ go tool vet . 2>&1 | grep -vE 'Godeps|unreachable code' | wc -l | awk '{if ($$1 >0) {exit 1}}'

CI use the make exit code to determine whether the check is OK. But if you use @! it will swallow the err exit code.